job-management/src/main/java/JobSubmission.java

```diff
@@ -29,8 +29,8 @@
 import static com.hazelcast.jet.pipeline.JournalInitialPosition.START_FROM_OLDEST;
 
 /**
- * We demonstrate how a job can be submitted to Jet
- * and further managed via the {@link Job} interface.
+ * We demonstrate how a job can be submitted to Jet and further managed via the
+ * {@link Job} interface.
  */
 public class JobSubmission {
```

Comment:
would be better to rename this to JobSuspendResume 

Submission doesn't really mean much